### PR TITLE
adding error log tool

### PR DESCRIPTION
**Pull request by tareddingatbaici**
Added log viewer from here: https://github.com/amol-anand/tools/tree/main/tools/log-viewer

This is a tool, you access it by going to http://localhost:3000/tools/log-viewer/index.html . Basically, it's a basic html page with some javascript that's stored on the server instead of google docs. When you first try to use it, it sends you to a json object with a list of login urls. I clicked the google one, and authenticated via google and was able to get the tool to work.

Some days might not have logs, so if you aren't getting any and you aren't getting an auth error, then you can try choosing a different start date

Other work I did: I set up [this](https://docs.google.com/spreadsheets/d/1J72xqoH4-o1NsLlaqmGOb1QDfkT5WEILgWAkHGybEQE/edit#gid=0) google sheet as per https://www.aem.live/docs/auditlog and https://www.aem.live/docs/configuration . This sheet controls group access, I added the ability with anyone with a blue acorn ici email to view the logs. If you make changes, just don't forget to publish the sheet like you would anything else.

### CODE REVIEW

File: head.html [hunk 1]
```diff
@@ -1,4 +0,0 @@
-<meta name="viewport" content="width=device-width, initial-scale=1"/>
```
Comment (jamesmcconnellblueacornici) [inline head.html]: Do we intend to remove this? It appears to break the page styles when doing so

Comment (tareddingatbaici) [reply head.html]: Oh that was my bad, I thought it wasn't in use because the head was being loaded from the drive. I'll add it back, one sec

Comment (tareddingatbaici) [reply head.html]: Ok should be good now

Comment (jamesmcconnellblueacornici) [reply head.html]: thanks!

**Review by jamesmcconnellblueacornici — APPROVED** [review]
reapproved +1

**Pull request merged**